### DISCUSSION TITLE
Member-Only TriliumNext Organization Readme

### DISCUSSION THREAD

**Discussion by meichthys** [TriliumNext/Trilium#6065] [Releases]
I've added a [member-only TriliumNext readme](https://github.com/TriliumNext?view_as=member) that we plan to use to document which of our members are our [github team](https://github.com/orgs/TriliumNext/teams) leads (As per our initial meeting on 2024.07.07, we are seeking two members to be leads for each [github team](https://github.com/orgs/TriliumNext/teams)). If you are in a github team and would like to be a team lead, please let me know. 

Also, if you are a team lead and would like a dedicated matrix room for synchronous communication, let me know and I can create a room and add it to our [Matrix space](https://matrix.to/#/#triliumnext:matrix.org).

**Comment by CrO2Cl2** [comment]
If noone is against it, I would volunteer to be one of the team-leads of the issue triaging team

**Reply by CobriMediaJulien** [reply]
Go for it. We call it leads but it´s more a way to improve communication and collaborative work regarding information flow.

**Reply by meichthys** [reply]
@CrO2Cl2 I've added you to the member-only readme! Thanks for steping up!
